roles/mongodb-common/tasks/main.yml

```diff
@@ -46,6 +55,12 @@
 - name: start mongodb service
   service: name=mongod state=started enabled=True
 
+- name: install pymongo for mongodb_user module
```

Comment:
both ubuntu and rhosp need this.
previous ursula version, this only happen on mongodb primary server, since only primary node need mongodb_user module.
For this release mongodb secondary and arbiter need mongodb_replication module , so need to install pymongo too.
So I move this pymongo installation from primary only to mongodb-common.